### DISCUSSION TITLE
Set default currency on Money field to stores base currency

### DISCUSSION THREAD

**Discussion by rynpsc** [craftcms/cms#11437] [Ideas] [answered]
A nice quality of improvement to the Money field would be to default to the stores base currency when Craft Commerce is installed.

**Comment by brandonkelly** [comment, marked as answer]
You mean within the field settings?

We don’t have any other places where Craft makes a special case in the logic if a plugin is installed, and even if we did make an exception in this case, it would be pretty short lived since Commerce 5 is going to get multi-store support, and each store will have it own base currency.
